docs/content/3.manual/manual.yml

```diff
@@ -408,15 +412,16 @@ sections:
           Like `.[]`, but no errors will be output if . is not an array
           or object.
 
-      - title: "`,`"
+      - title: "Comma, Output Stream Join: `,`"
```

Comment:
`Output Stream:` is simpler. Can remove `comma` b/c it's redundant. Would avoid referring to it as a "join", since it's actually a FORK opcode and will will thus come across very much as a "split". The steaming aspect is important. Would focus on that instead.